src/main/java/redis/clients/jedis/executors/CommandExecutor.java

```diff
@@ -6,6 +6,10 @@ public interface CommandExecutor extends AutoCloseable {
 
   <T> T executeCommand(CommandObject<T> commandObject);
 
+  default <T> T executeCommandToReplica(CommandObject<T> commandObject) {
+    return executeCommand(commandObject);
+  }
+
```

Comment:
Let's start with keep this feature only specific to `ClusterCommandExecutor`.
```suggestion
```

Reply:
done